apps/sla/models.py

```diff
@@ -66,25 +76,40 @@ def parse_date(date_desc):
             """
             return date_desc.lower().strip().replace(" ", "_").replace("_date", "_dt")
 
-        def parse_type(type_desc):
+        def parse_type_as_str(string_type_desc: str):
             """
             translate human-readable description into the actual
             TemporalPolicy type and its optional ending
             """
             ending = cls.EndingTypes.ANY_DAY
             for ending_type, _ in cls.EndingTypes.choices:
-                if ending_type in type_desc:
+                if ending_type in string_type_desc:
                     ending = ending_type
-                    type_desc = type_desc.replace(ending_type, "").strip()
+                    string_type_desc = string_type_desc.replace(ending_type, "").strip()
```

Comment:
My personal preference would be not to rename these variables so the git history is kept intact. We are not changing anything else anyway. But I do not insist on it.